src/plugins/navigation_embeddable/public/components/editor/navigation_embeddable_editor.scss

```diff
@@ -1,4 +1,4 @@
-@import '../mixins';
+@import '../../mixins';
 
 .navEmbeddablePanelEditor {
   max-inline-size: $euiSizeXXL * 18; // 40px * 18 = 720px
```

Comment:
Oh, good catch!! My math was off because I was accidentally using the programmer calculator, which rounds to the nearest integer - so the `min-width` was off by `8px` 😆 

Resolved with 455a5552b6251e96b1bcd730101594680b6b0dbc